app/presenters/setting_presenter.rb

```diff
@@ -7,33 +7,14 @@ class SettingPresenter
   attribute :id, :integer
   attribute :category, :string, default: 'Setting::General'
   attribute :name, :string
-  attribute :default
   attribute :value
+  attribute :default
```

Comment:
I'm guessing this order change is by accident, can you revert it for a cleaner history?